### PR TITLE
feat: improve departments view page

### PR DESCRIPTION
**Pull request by AfonsoMartins26**
Hides collaborators table when there are no collaborators

### CODE REVIEW

File: lib/atomic_web/live/department_live/show.html.heex [hunk 1]
```diff
@@ -54,48 +54,50 @@
...
+                      <td class="whitespace-nowrap px-3 py-4 text-sm text-zinc-500"><%= display_date(collaborator.inserted_at) %> <%= display_time(collaborator.inserted_at) %></td>
+                      <td class="whitespace-nowrap px-3 py-4 text-sm text-zinc-500"><%= display_date(collaborator.updated_at) %> <%= display_time(collaborator.updated_at) %></td>
+                      <td class="relative whitespace-nowrap py-4 pl-3 pr-4 text-right text-sm font-medium sm:pr-6">
+                        <.link patch={Routes.department_index_path(@socket, :edit, @organization, @department)} class="button">
+                          <div type="button" class="inline-flex justify-center py-2 px-4 w-fit text-sm font-medium text-zinc-700 bg-white rounded-md border border-zinc-300 shadow-sm hover:bg-zinc-50" id="sort-menu-button" aria-expanded="false" aria-haspopup="true">
+                            <Heroicons.pencil solid class="mr-3 w-5 h-5 text-zinc-400" /> Edit
+                          </div>
+                        </.link>
```
Comment (MarioRodrigues10) [inline lib/atomic_web/live/department_live/show.html.heex:90]: Looks like this lines aren't doing anything, could you remove them.
```suggestion

```

**Review by MarioRodrigues10 — DISMISSED** [review]
Looks awesome, just make those changes and it's ready to go :star_struck:

**Review by ruilopesm — APPROVED** [review]
Great work! 🧡

**Review by MarioRodrigues10 — APPROVED** [review]
Looks good to me :rocket: :orange_heart:

**Review by joaodiaslobo — APPROVED** [review]
LGTM! 🧡

**Pull request merged**